### PR TITLE
CCloudify uncaught exception handler

### PR DESCRIPTION
**Pull request by awalther28**
### Description

Convert uncaught exception handler. Manually stepped through the Confluent steps. I have verified that the test passes locally and in semaphore.

### Staging Docs
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/convert-uncaught/error-handling/confluent.html

### CODE REVIEW

File: _includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+////
+  This is a sample content file for how to include a console consumer to the tutorial, probably a good idea so the end user can watch the results
+  of the tutorial.  Change the text as needed.
+
+////
+
+Now that you've ran the Kafka Streams application, it should have shut it self down due to reaching the max-error threshold.
```
Comment (ybyzek) [inline _includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc:7]: Minor (change to be made here and in the corresponding adoc in kstreams/markup)

```suggestion
Now that you've ran the Kafka Streams application, it should have shut itself down due to reaching the max-error threshold.
```

**Review by ybyzek — APPROVED** [review]
Minor comments and then LGTM, thanks @awalther28

File: _includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+LEAD
+TO
+CONFLUENT
+ALL
+STREAMS
+LEAD
+TO
+CONFLUENT
+GO
+Processed a total of 12 messages
```
Comment (ybyzek) [inline _includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc:30]: This doesn't stop at 12, does it?  It should be `ctrl-c` to stop.

Comment (awalther28) [reply _includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc:30]: Yup, it's there on line 36 :) Do you want me to move it up?

Comment (awalther28) [reply _includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc]: Great catch! There was already `crtl-c` exit text but I failed to remove the "Processed a total of 12 messages". I removed that text in commit https://github.com/confluentinc/kafka-tutorials/pull/854/commits/e33b3de9bc1b607c46a6a9827339ef5ce353645e

**Pull request merged**